src/stores/models/AddressBalance.test.js

```diff
@@ -0,0 +1,54 @@
+import AddressBalance from './AddressBalance';
+import { decimalToSatoshi } from '../../helpers/utility';
+
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  clear: jest.fn(),
+};
+const navigatorMock = {};
+global.navigator = navigatorMock;
+global.localStorage = localStorageMock;
+
+describe('stores/models/AddressBalance', () => {
+  let ab;
+  beforeEach(() => ab = null);
+  it('Constructor input Satoshi', () => {
+    const input = {
+      address: '123456',
+      qtum: decimalToSatoshi(new Number(10)), // eslint-disable-line
```

Comment:
`new Number()` not necessary unless you have a reason to?

Reply:
I personally just like to give 1 or 2 dumb cases when I test, very rare but occasionally they do help lol.
If you think I really should remove it, I have no problem.

Reply:
Yes please remove any non-realistic tests